bootstrap/kubeadm/internal/controllers/kubeadmconfig_controller.go

```diff
@@ -321,12 +323,40 @@ func (r *KubeadmConfigReconciler) refreshBootstrapToken(ctx context.Context, con
 		return ctrl.Result{}, err
 	}
 
-	log.Info("Refreshing token until the infrastructure has a chance to consume it")
-	if err := refreshToken(ctx, remoteClient, token, r.TokenTTL); err != nil {
```

Comment:
Moving code out of `refreshToken` because it's only used once and I didn't see the point of having two functions called "refresh token if necessary".